websphere-liberty/content.md

```diff
@@ -0,0 +1,106 @@
+# Overview 
+
+This image contains IBM WebSphere Application Server for Developers Liberty
+Profile and the IBM Java Runtime Environment. For more information on WebSphere
+Application Server Liberty Profile, see the [WASdev][getting-started] site.
+
+# Usage
+
+In order to use the image, it is necessary to accept the terms of the WebSphere
+Application Server for Developers and IBM JRE licenses. This is achieved by
+specifying the environment variable `LICENSE` equal to `accept` when running the
+image. You can also view the license terms by setting this variable to
+`view`. Failure to set the variable will result in the termination of the
+container with a usage statement.
+
+The image is designed to support a number of different usage
+patterns. The following examples are based on the Liberty [application
+deployment sample][sample] and assume that
+[DefaultServletEngine.zip][zip] has been extracted to `/tmp` and the
+`server.xml` updated to accept HTTP connections from outside of the
+container by adding the following element inside the `server` stanza:
+
+```
+<httpEndpoint host="*" httpPort="9080" httpsPort="-1"/>
+```
```

Comment:
Can we change all the code blocks to be 4 space indented, the docker hub has issues displaying the triple back-tick blocks (see [rawdns](https://registry.hub.docker.com/u/tianon/rawdns/) vs [github](https://github.com/tianon/rawdns))